app.yaml

```diff
@@ -59,6 +59,13 @@ handlers:
   static_files: core/templates/dev/head/\1
   upload: core/templates/dev/head/(.*\.(js))$
   secure: always
+- url: /templates/dev/head/components/(.*\.(html))$
+  static_files: core/templates/dev/head/components/\1
+  upload: core/templates/dev/head/components/(.*\.(html))$
+  # Remove application_readable after all components are referenced
```

Comment:
Formatting of TODOs should be as follows:

```
TODO(vojtechjelinek): Remove ...
```

The name in parentheses is the person who can explain what the TODO means.